.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "crates/upstream"]
 	path = crates/openvino-sys/upstream
 	url = https://github.com/openvinotoolkit/openvino
-	depth = 1
+	shallow = true
```

Comment:
https://git-scm.com/docs/gitmodules#Documentation/gitmodules.txt-submoduleltnamegtshallow

It's possible that `depth = 1` was working as expected, but it's not a documented property in `.gitmodules`.